### ISSUE TITLE
test-main.mtt template doesn't work

### ISSUE THREAD

**Issue by SlawomirMazgaj** [massiveinteractive/MassiveUnit#108]
Hello,
I'm able to successfully use `js_runner-html.mtt` and `swf_runner-html.mtt` but recently I wanted to enable JUnitReportClient in my `TestMain`. Because according to comment it's "Auto generated Test Application." I wanted to update template for it so I got it from [HERE](https://github.com/massiveinteractive/MassiveUnit/blob/master/tool/template/test-main.mtt). Unfortunately adding/removing/commenting anything in template file doesn't have any impact on generated `TestMain.hx`.

Summing up:
- .munit file has
`templates=munit/templates`

- test-main.mtt is inside `munit/templates` folder (js_runner-html and swf_runner-html are also here and they are working)

- I'm removing TestMain.hx file before each build just to be sure

Am I forgetting about something?
